src/plone/restapi/tests/test_documentation.py

```diff
@@ -608,3 +608,23 @@ def test_documentation_navigation(self):
         response = self.api_session.get(
             '{}/@components/navigation'.format(self.document.absolute_url()))
         save_request_and_response_for_docs('navigation', response)
+
+    def test_documentation_principals(self):
```

Comment:
@sneridagh ah, now I get it. Maybe we should use a query (or no query at all) here that returns at least one user.